frontend/libs/studio-components/src/components/StudioBlobDownloader/StudioBlobDownloader.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useMemo } from 'react';
+import { BlobDownloader } from '@studio/pure-functions';
+import { StudioButton } from '@studio/components';
+
+export type StudioBlobDownloaderProps = {
+  data: string;
+  fileName: string;
+  fileType?: string;
+  linkText: string;
+};
+
+export const StudioBlobDownloader = ({
+  data,
+  fileName,
+  fileType = 'application/json',
+  linkText,
+}: StudioBlobDownloaderProps) => {
+  const blobDownloader = useMemo(
+    () => new BlobDownloader(data, fileType, fileName),
+    [data, fileType, fileName],
+  );
+  const handleExportClick = () => {
+    blobDownloader.handleDownloadClick();
+  };
+
+  return (
+    <StudioButton onClick={handleExportClick} variant='tertiary'>
```

Comment:
Since this component extends StudioButton, we should inherit the props from StudioButton and pass them down using ...rest. What do you think? 😊 

```suggestion
export type StudioBlobDownloaderProps = {
  data: string;
  fileName: string;
  fileType?: string;
  linkText: string;
} & StudioButtonProps;

export const StudioBlobDownloader = ({
  data,
  fileName,
  fileType = 'application/json',
  linkText,
  rest
}: StudioBlobDownloaderProps) => {
  const blobDownloader = useMemo(
    () => new BlobDownloader(data, fileType, fileName),
    [data, fileType, fileName],
  );
  const handleExportClick = () => {
    blobDownloader.handleDownloadClick();
  };

  return (
    <StudioButton {..rest} onClick={handleExportClick} variant='tertiary'>
```